spec/factories/items.rb

```diff
@@ -37,5 +35,12 @@
       end
     end
 
+    trait :rejected do
+      state              'rejected'
```

Comment:
Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.